### ISSUE TITLE
일반인들을 위한 페이지를 만들 필요가 있을 것 같습니다

### ISSUE THREAD

**Issue by cheese98** [oleneyl/maplestory_dpm_calc#45]
DPM표 공개 후 여러 사이트들 돌아다니면서 반응을 확인해봤습니다. 반응이 폭발적인 것에 비해 정작 여기까지 피드백이 들어오지 않는 경향이 많이 보입니다. 가끔 지적하는 글이 올라와도 거의 커뮤니티에 올라오지, 여기까지 오는 경우는 많지 않은 것 같네요. 오픈소스이지만 소스코드를 읽지 못하는 대부분의 사람들은 내부 구조를 알 수 없다는 것이 원인인 것 같습니다.

이러한 이유로, 메이플스탯 사이트나 깃허브 저장소 위키 등의 장소에서 소스코드를 해설해주는 페이지를 만드는 게 좋을 것 같다고 생각합니다. 제작자분도 일일히 커뮤니티 돌아다니면서 고생할 필요가 줄어들 것이고, 일반 유저들도 프로그램 구조를 쉽게 이해할 수 있을 겁니다.

이렇게 말은 했지만 제가 요즘 좀 바빠서 바로 참여하기는 힘들 것 같습니다. 개인적인 일들 끝나는대로 프로젝트에 다시 참여하겠습니다.

**Comment by oleneyl** [comment]
네 해석이 딸려 있는 정보제공 페이지를 만들 필요가 충분히 있는 것 같습니다. Github Wiki를 활용하는 것이 제일 무난해 보입니다. 소스 코드에 삽입하는 방법이 제일 좋겠지만 접근성이 너무 떨어질 것 같네요.
시간 날때 틈틈이 작업해 보도록 하겠습니다. 소스코드 재검토 하는 겸 해서 Wiki를 하나 생성해야 겠네요.

**Comment by Gwang-jun** [comment]
두 분이서 대부분의 기여를 하신 것 같은데 정말 수고하셨고 수고하십니다.. 저도 전공자가 아니라서 단편적으로만 코드를 이해하는데 이걸 두 분이서 모든 직업에 대해서 하는 건 쉽지 않아보입니다.
하지만 사공이 많으면 배가 산으로 가는 법이라고 생각합니다. 40여개 직업에 각기 다른 이해도를 가진 기여자가 몰려드는게 (물론 지금보다야 낫겠지만) 바람직한지는 모르겠습니다. 아무래도 각 직업이나 직업군별로 홍보를 해서 코드를 이해하시고 직업 자체도 잘 이해하시는 분들이 코드를 해설해서 각 직업 게시판에 공론화시키도록 유도하는게 낫지 않을까요? 그럼 다른 분들이 의견을 주면 그걸 다시 직업별로 취합하고 간소화해서 가져오도록 하는게 낫지않나 싶습니다.

**Comment by cheese98** [comment]
@Gwang-jun 
아쉽지만 코드 이해도랑 직업 이해도가 둘 다 높은 분을 찾기는 힘들 것 같습니다. 일단 코드를 읽을 줄 아는 사람들이 한줄씩 일일히 해석해주는 게 가장 좋은 방법이지 않을까 싶습니다. 해당 직업 유저들이 잘 이해하고 피드백을 내준다면 인지도는 알아서 올라갈 겁니다.

**Comment by cheese98** [comment]
[제 포크 쪽에 가이드 초안을 올려봤습니다.](cheese98/maplestory_dpm_calc/wiki/모든-사람들을-위한-가이드-(작성중)) 제가 시간이 별로 없어서 얼마 못 썼습니다. 아직 갈 길이 머네요.
